eradiate/util/xarray.py

```diff
@@ -656,7 +666,10 @@ def validate_metadata(self, dataset_spec, normalize=False):
         # Validate dataset metadata
         if dataset_spec.title is not None:
             dataset.attrs = validate_metadata(
-                dataset, dataset_spec, normalize=normalize
+                data=dataset,
+                spec=dataset_spec,
+                normalize=normalize,
+                allow_unknown=dataset_spec.allow_unknown
```

Comment:
You can simply do `allow_unknown=allow_unknown` and make `allow_unknown` a parameter of `EradiateDatasetAccessor.validate_metadata()`. You should then propagate this behaviour to `EradiateDataArrayAccessor.validate_metadata()` as well.